### ISSUE TITLE
Doesn't build. Podfile has external references.

### ISSUE THREAD

**Issue by GreenRays** [restjohn/mgrs-overlay-mapkit#1]
pod 'GEOTRANS', :path => '../../GEOTRANS'
pod 'GEOTRANSUtil', :path => '../../GEOTRANSUtil'

**Comment by restjohn** [comment]
Indeed!  That project has not seen any attention in quite some time, unfortunately, and was only a work in progress.  I might get back to it but quite honestly not for at least a few months.
